mysql_con.js

```diff
@@ -2,6 +2,9 @@ var mysql           = require("mysql");
 var express         = require("express");
 var app             = express();
 var bodyParser      = require("body-parser");
+var crypto          = require("crypto");
+var gets            = require("./app/mysql/get.js");
+var posts           = require("./app/mysql/post.js");
 
```

Comment:
Because I was looking for a return of the number of occurrences of what the user has entered being matched by user emails and passwords in the database, I couldn't use a GET command, because it wouldn't accept the data field in the jquery-Ajax call for me to specify the query. 

I'd originally attempted to use them as a GET command, but it wouldn't take, so I made them POSTs instead (copy/pasted to the other file). I left the connection there because I'll need it later anyways.